packages/shared/src/lib/invitations/index.ts

```diff
@@ -37,7 +45,16 @@ export const SendUserInvitationRequest = Type.Union([
         type: Type.Literal(InvitationType.PROJECT),
         email: Type.String(),
         projectId: Type.String(),
-        projectRole: Type.Enum(ProjectMemberRole),
+        projectRole: Type.Object({
+            id: Type.String(),
+            created: Type.String(),
+            updated: Type.String(),
+            name: Type.String(),
+            permissions: Type.Array(Type.String()),
+            platformId: Type.Optional(Type.String()),
+            type: Type.String(),
+            userCount: Type.Optional(Type.Number()),
+        }),
     }),
```

Comment:
You need to send the id not the whole project role Id.

It makes things harder for the api users (frontend), It also can be security issue If you trust the frontend, but I know we have relations but generally it's anti pattern.